test/views/sections/items/ItemProducerSizeViewSpec.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.sections.items
+
+import base.SpecBase
+import fixtures.messages.sections.items.ItemProducerSizeMessages.English
+import forms.sections.items.ItemProducerSizeFormProvider
+import models.GoodsTypeModel.{Beer, Spirits, Wine}
+import models.requests.DataRequest
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.i18n.Messages
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import views.html.sections.items.ItemProducerSizeView
+import views.{BaseSelectors, ViewBehaviours}
+
+class ItemProducerSizeViewSpec extends SpecBase with ViewBehaviours {
+
+  object Selectors extends BaseSelectors
+
+  "Item Producer Size view" - {
+
+    s"when being rendered in lang code of '${English.lang.code}'" - {
+
+      "when the goodsType is beer and the years are 2022 to 2023" - {
```

Comment:
I tested it against the three different versions of the content that we'll show for the page and with 3 different years just to be sure everything changed as it should since they're just passed through as 2 different strings. What was it you were thinking I should add?

Reply:
It's not a massive thing, maybe we can look after MVP